tests-build/README.md

```diff
@@ -6,5 +6,5 @@ To run all of the tests in this directory, run the following commands:
 cargo test --features full
 cargo test --features rt
 ```
-If one of the tests fail, you can pass `TRYBUILD=overwrite` to the `cargo test`
+If one of the tests fails, you can pass `TRYBUILD=overwrite` to the `cargo test`
```

Comment:
Thanks for catching this. Actually, I would prefer to change it to plural instead:
```suggestion
If any of the tests fail, you can pass `TRYBUILD=overwrite` to the `cargo test`
```

Reply:
Corrected!